### PR TITLE
added sticker::get_sticker_url getter

### PR DESCRIPTION
**Pull request by Commandserver**
https://discord.com/developers/docs/reference#image-formatting-cdn-endpoints

### CODE REVIEW

File: include/dpp/message.h [hunk 1]
```diff
@@ -839,6 +839,13 @@ struct DPP_EXPORT sticker : public managed {
 	 */
 	std::string build_json(bool with_id = true) const;
 
+	/**
+	 * @brief Get the sticker url
+	 *
+	 * @return std::string icon url
+	 */
+	std::string get_sticker_url() const;
```
Comment (braindigitalis) [inline include/dpp/message.h:847]: wouldn't `get_url()` be a better method name, as its obvious what url you're getting - its a method of the `sticker` object 😃

**Review by braindigitalis — CHANGES_REQUESTED** [review]

File: include/dpp/message.h (hunk 1 above)
Comment (Commandserver) [reply include/dpp/message.h:847]: i agree but i tried to follow the current style of all the other url getters

Comment (Lmvsk1) [reply include/dpp/message.h:847]: 1

Comment (Lmvsk1) [reply include/dpp/message.h:847]: 2

**Comment by Commandserver** [comment]
im gonna add a parameter to choose whether to accept json format

**Pull request merged**